### PR TITLE
Add `GetCenter` to `ControlTestBase` to avoid duplicated code

### PR DESCRIPTION
**Pull request by elachlan**
Adds `GetCenter` to `ControlTestBase` in `System.Windows.Forms.UI.IntegrationTests` to avoid duplicated code.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/winforms/pull/8453)

### CODE REVIEW

**Comment by elachlan** [comment]
CC: @Tanya-Solyanik

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs [hunk 1]
```diff
@@ -296,5 +296,12 @@ protected async Task RunFormWithoutControlAsync<TForm>(Func<TForm> createForm, F
         internal struct VoidResult
         {
         }
+
+        internal static Point GetCenter(Rectangle cell)
+        {
+            return new Point(GetMiddle(cell.Right, cell.Left), GetMiddle(cell.Top, cell.Bottom));
+
+            static int GetMiddle(int a, int b) => (a + b) / 2;
```
Comment (dmitrii-drobotov) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs:304]: Theoretically `a + b` could overflow the `int` range, that's why other versions used `a + ((b - a) / 2)`. But I suppose we don't have to worry about such values in tests?

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs [hunk 2]
```diff
@@ -296,5 +296,12 @@ protected async Task RunFormWithoutControlAsync<TForm>(Func<TForm> createForm, F
         internal struct VoidResult
         {
         }
+
+        internal static Point GetCenter(Rectangle cell)
```
Comment (dmitrii-drobotov) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs:300]: Can it also be used here?
https://github.com/dotnet/winforms/blob/171fb92b3aca2d73a377e82daa47e42c297e5b10/src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/DragDropTests.cs#L208-L210

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs (hunk 1 above)
Comment (elachlan) [reply src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/Infra/ControlTestBase.cs:304]: I'll update it.

File: src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/OpenFileDialogTests.cs [hunk 3]
```diff
@@ -32,5 +32,28 @@ public void OpenFileDialogTests_OpenWithExistingInitDirectory_Success()
             dialog.InitialDirectory = Path.GetTempPath();
             Assert.Equal(DialogResult.Cancel, dialog.ShowDialog(dialogOwnerForm));
         }
+
+        [WinFormsFact]
+        public void OpenFileDialogTests_Open()
```
Comment (dmitrii-drobotov) [inline src/System.Windows.Forms/tests/IntegrationTests/UIIntegrationTests/OpenFileDialogTests.cs:37]: This test and `FileDialog` changes seem to be related to another issue?

**Review by dmitrii-drobotov — APPROVED** [review]

**Review by dreddy-work — APPROVED** [review]

**Pull request merged**